### ISSUE TITLE
Setter methods for subreddit settings

### ISSUE THREAD

**Issue by pl00h** [reddit/devvit#62]
It would be useful to have the ability to set subreddit settings (and the type and other info that we can currently get but not set).

**Comment by Yay295** [comment]
I believe this is somewhat related to #31.

**Comment by fsvreddit** [comment]
@pl00h I don't think this is a duplicate, #31 isn't related to the ability to set subreddit settings from code
